html/donate/en.html

```diff
@@ -0,0 +1,98 @@
+
+<!doctype html>
+<html class="no-js" lang="fr">
+<head>
+<meta charset="utf-8">
+
+<title>Donate to Open Food Facts</title>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<meta property="fb:app_id" content="219331381518041">
+<meta property="og:type" content="food">
+<meta property="og:title" content="">
+<meta property="og:url" content="https://world.openfoodfacts.org/donate-to-openfoodfacts">
```

Comment:
This URL is 404

Reply:
I think that's because it hasn't been deployed to production yet.